cdisc_rules_engine/services/reporting/excel_report.py

```diff
@@ -59,6 +60,23 @@ def get_export(
         )
         wb["Conformance Details"]["B3"] = f"{round(self._elapsed_time, 2)} seconds"
         wb["Conformance Details"]["B4"] = __version__
+
+        # write dataset metadata
+        datasets_data = [
+            [
+                dataset.get("filename"),
+                dataset.get("label"),
+                dataset.get("full_path"),
```

Comment:
this currently works on windows, but the full path name is not ideal. It looks like this:
`C:/Users/GerryCampion/Code/cdisc-rules-engine-tests/msg\ae.xpt`
If you change this line:
https://github.com/cdisc-org/cdisc-rules-engine/blob/f64c496a1f8e900fd0c274ad9effa295ce1ae427/core.py#LL197C12-L197C12
to this (using Pathlib):
`str(Path(data).joinpath(fn))` 
(or even maintain the Path type throughout and convert it to string later), then I get a path like this in the report on windows:
`C:\Users\GerryCampion\Code\cdisc-rules-engine-tests\msg\ae.xpt`

Reply:
updated